gpii/node_modules/flowManager/src/BrowserChannel.js

```diff
@@ -34,7 +34,7 @@
 
                 gpii.request.flowManager.logAndNotify("Lifecycle manager returned: ", events.onError)(errMsg);
             } else {
-                gpii.settingsHandlers.webSockets.addClient(solutionId, fluidRequest.socket);
+                gpii.settingsHandlers.webSockets.addClient(solutionId, fluidRequest.socket, entries[solutionId].settingsHandlers[0].options);
```

Comment:
Ok,

then I will update it according to the new format. Will ping you when it's ready again for review